### ISSUE TITLE
<html> element does not have a lang attribute

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-services/screenshots#5015]
[detailed info on html-has-lang rule](https://dequeuniversity.com/rules/axe/3.1/html-has-lang?application=axeAPI)
```
  Rule Violated:
  html-has-lang - Ensures every HTML document has a lang attribute
  	URL: https://dequeuniversity.com/rules/axe/3.1/html-has-lang?application=axeAPI
  	Impact Level: serious
  	Tags: cat.language wcag2a wcag311
  	Elements Affected:
  	1) Target: html
  		The <html> element does not have a lang attribute
```

**Comment by yehudab** [comment]
Since I’m already touching the `<html>` tag in #4261, I can easily handle this issue as well.
